pre_commit_hooks/check_executables_have_shebangs.py

```diff
@@ -21,7 +22,9 @@ def zsplit(s: str) -> List[str]:
 
 
 def check_executables(paths: List[str]) -> int:
-    if sys.platform == 'win32':  # pragma: win32 cover
+    if (
+        sys.platform == 'win32' or 'microsoft' in platform.uname()[3].lower()
```

Comment:
can you share an example output here and/or add a test?  I don't have access to WSL to verify this change

Reply:
Should I add a comment in the code with the sample output?
I'm not sure what kind of test to add for this, mock the output of `platform.uname()` and assert `_check_git_filemode` is called?

Sample output:
```
$ python3
Python 3.7.5 (default, Nov  7 2019, 10:50:52)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.uname()
uname_result(system='Linux', node='DESKTOP-A7438J1', release='4.4.0-19041-Microsoft', version='#1-Microsoft Fri Dec 06 14:06:00 PST 2019', machine='x86_64', processor='x86_64')
>>> platform.uname()[3].lower()
'#1-microsoft fri dec 06 14:06:00 pst 2019'
>>>
```

Reply:
@asottile Is the sample output enough, and if not, is the suggested test case acceptable?

Reply:
this check seems much too weak imo